### ISSUE TITLE
NTR: Tuberculosis skin test positive (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8733]
New term request Tuberculosis skin test positive (samant_101122001619): Tuberculin, Mantoux; PPD (purified protein derivative)  UMLS: 1406019

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000925
- label: Tuberculin skin test
- parents: [Positive infectious agent test (HP:0430030)](https://hpo.jax.org/app/browse/term/HP:0430030)
- pmid: 
- def: A positive result of the tuberculin skin test suggesting previous expouse or infection with mycobacterium teberculosis. A standard dose of five tuberculin units (TU) (0.1ml) is injected intradermally (into the skin) and read 48 to 72 h later. A person who has been exposed to the bacteria is expected to mount an immune response in the skin containing the bacterial proteins. The volar aspect of the forearm is the preferred site of test. The basis of reading is the presence or absence of induration, which may be determined by inspection (from a side view against the light as well as by direct light) and by palpation. For standardization, the diameter of induration should be measured transversely to the long axis of the forearm and recorded in millimeters.
- comment: The tuberculin most widely used is purified protein derivative (PPD), which is derived from cultures of M. tuberculosis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8733
- synonyms: Positive Mantoux tuberculosis test(NONE)|Tuberculosis skin test positive(NONE)

**Comment by MickeySegal** [comment]
Shouldn't this be "Positive tuberculin skin test"?

**Comment by pnrobinson** [comment]
@MickeySegal 
thanks for pointing out this oversight. 
I have corrected the primary label to

Positive tuberculin skin test
